### PR TITLE
DOC: implement general documentation sections

### PR DESCRIPTION
**Pull request by jorgepiloto**
Solves #9.

### CODE REVIEW

File: doc/source/getting_started.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Installation
+============
+
+This package may be installed following two procedures: either the pip package
+manager installation or the manual installation. The process to be followed for
+each of them is shown in the upcoming sections.
+
+The ``pyansys-tools-versioning`` package currently supports Python >=3.7 on
+Windows, Mac OS, and Linux.
```
Comment (RobPasMue) [inline doc/source/getting_started.rst:12]:
```suggestion
Windows, macOS, and Linux.
```

File: doc/source/getting_started.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+   python -m pip install git+https://github.com/pyansys/pyansys-tools-versioning.git
+
+For a local development version, install with (requires pip >= 22.0):
+
+.. code-block:: bash
+
+   git clone https://github.com/pyansys/pyansys-tools-versioning.git
+   cd pyansys-tools-versioning
+   python -m pip install -e .
+
```
Comment (RobPasMue) [inline doc/source/getting_started.rst:33]: I think that when dealing with ``pyproject.toml`` projects, you cannot install it in editable mode, right?

File: doc/source/getting_started.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+For example, on Linux with Python 3.7, unzip it and install it with the
+following:
+
+.. code-block:: bash
+
+   unzip pyansys-tools-versioning-v<major.minor.patch>-wheelhouse-Linux-3.7.zip wheelhouse
+   python -m pip install pyansys-tools-versioning -f wheelhouse --no-index --upgrade --ignore-installed
+
+If you’re on Windows with Python 3.9, unzip to a wheelhouse directory and
```
Comment (RobPasMue) [inline doc/source/getting_started.rst:54]:
```suggestion
If you are on Windows with Python 3.9, unzip to a wheelhouse directory and
```

File: doc/source/user_guide.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        def old_method(self):
+            pass
+
+        @requires_version("0.5.1", VERSION_MAP)
+        def new_method(self):
+            pass
+
+
+Suppose that two servers are created using previous class. Each of the servers
+is using a different version, meaning that some of the methods are be available
```
Comment (RobPasMue) [inline doc/source/user_guide.rst:50]:
```suggestion
is using a different version, meaning that some of the methods are available
```

**Review by RobPasMue — APPROVED** [review]
LGTM @jorgepiloto, some minor comments! Thanks for the effort =)

File: doc/source/getting_started.rst (hunk 2 above)
Comment (jorgepiloto) [reply doc/source/getting_started.rst:33]: There used to be a problem with this some years ago. However, `pip` latest releases support editable installs for packages using a `pyproject.toml`.

Comment (RobPasMue) [reply doc/source/getting_started.rst:33]: I guess I need to update my ``pip`` 😄

**Pull request merged**